zerver/tests/test_event_queue.py

```diff
@@ -674,6 +675,28 @@ def test_bot_recipient(self) -> None:
         self.destroy_event_queue(hambot, self.client_descriptor.event_queue.id)
         self.client_descriptor = hamlet_client_descriptor
 
+    # Internal PMs
+    def test_disable_external_notifications(self) -> None:
+        # Email and push notifications should not be sent.
+        # Currently, external notifications are disabled for PMs sent by welcome bot.
+        # In general, it should be true for any sender, with the parameter set to true.
```

Comment:
This comment is confusing -- welcome bot is no longer relevant to what we're testing here.

# The disable_external_notifications parameter, used for messages sent by welcome bot, should
# result in no email/push notifications being sent regardless of the message type.